lib/actions-oclif/device/move.ts

```diff
@@ -144,8 +144,12 @@ export default class DeviceMoveCmd extends Command {
 		}
 
 		for (const uuid of params.uuid.split(',')) {
-			await balena.models.device.move(uuid, tryAsInteger(application));
-			console.info(`${uuid} was moved to ${application}`);
+			try {
+				await balena.models.device.move(uuid, tryAsInteger(application));
+				console.info(`${uuid} was moved to ${application}`);
+			} catch (err) {
+				console.info(`${err.message}, uuid: ${uuid}`);
```

Comment:
Nice! 👍   Additional easy improvement: add `process.exitCode = 1;` here, so that the CLI process exit code reflects the fact that something went wrong and some devices were not moved (in case the CLI is being scripted).

> I'd like to change method to use the following process: [...] Move devices if all device archs are valid.

You're welcome to, but my opinion/suggestion is that it is good enough as it is, and it could always be improved further in the future (in the way you suggest). I think it is already a great feature delivery as it is, and additional PRs to deliver the same to other commands (e.g. `device rm`) (or PRs for other issues) would offer a greater return on investment at this point! :-)

Reply:
`process.exitCode = 1` added. I've also got a branch started to add the same functionality to `device rm`.

Reply:
Great! Would you mind squashing the 4 commits in a single one? Probably using the description of your first commit. Generally we follow the principle of "meaningful commits", and the commits in this PRs at the moment are effectively amending each other.

Reply:
I debated about doing so but waited to see what was typically done for the project.

Reply:
Oh, sorry, just noticed one more thing: the `Change-type` for the commit should be `minor` rather than `patch`, as it is delivering a new feature (and one that is visible to users!).